src/ReactWrapper.jsx

```diff
@@ -175,6 +177,8 @@ class ReactWrapper {
     if (this.root !== this) {
       throw new Error('ReactWrapper::instance() can only be called on the root');
     }
+    if (!this.component.state.mount) return instanceMap.get(this.component);
```

Comment:
I wonder if this should be `return this.component.getInstance() || instanceMap.get(this.component);`, to cover "not yet mounted", and to always prioritize the component's getInstance method?